server/statistics/hot_peer_cache.go

```diff
@@ -60,6 +60,14 @@ var MinHotThresholds = [RegionStatCount]float64{
 	RegionWriteQueryNum: 32,
 }
 
+// DefaultThresholdsUpdateInterval is the default interval to update thresholds.
+var DefaultThresholdsUpdateInterval = StoreHeartBeatReportInterval * time.Second
```

Comment:
1. the `StoreHeartBeatReportInterval` can be adjusted by tikv config 
2. the refresh interval should be less than store hearbeat interval to keep the next calculate must use the latest threshold?

Reply:
1. I will make store heartbeat interval sync with tikv in another pr.
2. Good idea